Add find-opengl-enum extension

jsyishan opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

mil3na commented:
Hi!
Thank you for your contribution! My name is Milena and I will be reviewing it today.

- Could you please enter an icon for your extension that is not Raycast's icon?
- It would be interesting if selecting a number did something. Maybe copy the value to the clipboard?
- Does everything need the OpenGL title on the right?
- Searching by content like "BIT" would be a great add as well.

Do you mind taking a look at the points above?

jsyishan commented:
> Hi! Thank you for your contribution! My name is Milena and I will be reviewing it today.
> 
> * Could you please enter an icon for your extension that is not Raycast's icon?
> * It would be interesting if selecting a number did something. Maybe copy the value to the clipboard?
> * Does everything need the OpenGL title on the right?
> * Searching by content like "BIT" would be a great add as well.
> 
> Do you mind taking a look at the points above?

Hi Milena and thanks for your review!
* Sure I've replaced it with a proper one.
* The only functionality of the extension is just searching the enum, not aim to copy or paste something.
* Yes, it's the name of some different OpenGL distributions, so it shall be shown as title
* Yes, I have fixed that issue

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
